services/src/main/java/org/keycloak/protocol/saml/profile/ecp/authenticator/HttpBasicAuthenticator.java

```diff
@@ -52,17 +53,24 @@ public String getDisplayType() {
 
     @Override
     public String getReferenceCategory() {
-        return null;
+        return "basic";
     }
 
```

Comment:
No idea. But I don't think there's a problem with having docker specific HTTP Basic Authenticator. Maybe we should just call it DockerAuthenticator to avoid any confusion as to what it's there for.